pkg/endpoint/policy.go

```diff
@@ -240,6 +240,32 @@ func (e *Endpoint) regenerateConsumable(owner Owner) (bool, error) {
 	return true, nil
 }
 
+// Must be called with endpointsMU held
+func (e *Endpoint) regenerateL3Policy(owner Owner) (bool, error) {
+	c := e.Consumable
```

Comment:
No, since the caller in regenerate() calls e.regeneratePolicy() only if e.Consumable is not nil.